<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Map_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>511</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 15px; /* 모서리 둥글기 설정 */
    background-color:rgba(113, 199, 236, 0.9); /* 투명도 50%의 녹색 */
    color: white; /* 글자색 */
    font: bold 40px; /* 폰트 스타일 */
    padding: 6px; /* 패딩 */
}</string>
    </property>
    <property name="text">
     <string>가게 정보</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Road_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>600</y>
      <width>511</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 15px; /* 모서리 둥글기 설정 */
    background-color:rgba(113, 199, 236, 0.9); /* 투명도 50%의 녹색 */
    color: white; /* 글자색 */
    font: bold 40px; /* 폰트 스타일 */
    padding: 6px; /* 패딩 */
}</string>
    </property>
    <property name="text">
     <string>출구 안내</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>501</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
